site/src/atomics/spacing.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+- `<css-property-name>-<size>-<screensize>`. For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md)
+
+### Accepted property names
+
+- `margin`
+- `padding`
+- `<margin|padding>-<left|right|top|bottom>`
+- `<margin|padding>-inline`
+- `<margin|padding>-block`
+
+### Accepted sizes
+
+- `none, xs, s, m, l, xl, xxl, xxxl`
+
+### Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usages
+
+### Adding uniform spacing
+
+```html
+<div class="margin-xs border">XS</div>
+<div class="margin-s border">X</div>
+<div class="margin-m border">M</div>
+<div class="margin-l border">L</div>
+<div class="margin-xl border">XL</div>
+<div class="margin-xxl border">XXL</div>
+<div class="margin-xxxl border">XXXL</div>
+```
+
+### Adding spacing to a side
+
+```html
+<div class="margin-left-s margin-bottom-m padding-s border">Small left margin</div>
+<div class="margin-right-l margin-bottom-m padding-s border">Large right margin</div>
+<div class="padding-inline-xl border">Padding-inline</div>
+<div class="padding-block-xxl border">Padding-block</div>
+```
+
+### Adding spacing on a particular screen size
+
+```html
+<div
+	class="padding-left-s padding-left-l-tablet padding-left-xl-desktop padding-left-xxl-widescreen border"
+>
+	Padding at different screen sizes
+</div>
+```
+
+## Inventory
```

Comment:
@v-mingho You're definitely right, and I was wrong; this seems too long. I do want to find some solution so that folks can eventually parse through all our output, but we don't have to do it here. For now, perhaps we should remove this section go with what you've written above? It seems pretty good to me.

Reply:
Removed inventory section.